snowflake/pkg/utils/utils.go

```diff
@@ -0,0 +1,40 @@
+// Copyright 2022 Antrea Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package utils
+
+import (
+	"io"
+	"net/http"
+	"os"
+)
+
+// Download a file from the given url to the current directory
+func DownloadFile(url string, filename string) error {
```

Comment:
I think this function should use a `Context` object. Maybe do some unification with https://github.com/antrea-io/theia/blob/5a15bb85d51acfc6d240c0c2e21a502b16c4dc3c/snowflake/pkg/infra/manager.go#L92. You could improve this function and use it as part of the `downloadAndUntar` implementation?

Reply:
Sure, merged it into `downloadAndUntar` function